servers/xr/xr_positional_tracker.cpp

```diff
@@ -196,13 +196,10 @@ void XRPositionalTracker::set_input(const StringName &p_action_name, const Varia
 	}
 
 	if (changed) {
-		// store the new value
-		inputs[p_action_name] = p_value;
-
 		// emit signals to let the rest of the world know
 		switch (p_value.get_type()) {
 			case Variant::BOOL: {
-				bool pressed = p_value;
+				bool pressed = inputs[p_action_name];
```

Comment:
This changes behaviour, it previously used the new value, now it uses the old value